src/quo2/components/avatars/user_avatar/view.cljs

```diff
@@ -0,0 +1,56 @@
+(ns quo2.components.avatars.user-avatar.view
+  (:require [clojure.string :as string]
+            [quo2.components.avatars.user-avatar.style :as style]
+            [quo2.components.markdown.text :as text]
+            [react-native.core :as rn]
+            [react-native.fast-image :as fast-image]))
+
```

Comment:
there's much refactoring here, or just aligning with best practices? e.g view, style files etc?

Reply:
well, there's refactoring but not too much, it's mainly split the code in style and view files